forms/src/pages/SurveyCreationPage/components/form/AnswerSection/AnswerType/RadioAnswer.tsx

```diff
@@ -0,0 +1,41 @@
+import "./SingleAnswer.scss";
+import { useContext, useEffect, useState } from "react";
+import { useTranslation } from "react-i18next";
+import { iAnswer } from "../../../../../../interfaces/iAnswer";
+import { sectionsContext } from "../../../../../../context/sectionsContext";
+
+function RadioAnswer(props: {
+  answer: iAnswer;
+  index: number;
+  questionIndex: number;
+}) {
+  const sections = useContext(sectionsContext);
+
+  const { t } = useTranslation();
+  const [answer, setAnswer] = useState(props.answer.answer);
+
+  if (!answer || answer === "") setAnswer(t("newAnswer") as string);
```

Comment:
Are you sure this is what you want to do?
If no answer was provided, you're setting answer to be a string that's likely not one of the options for this type of question